magsnake.md

```diff
@@ -0,0 +1,131 @@
+---
+name: "@magsnake"
+project: "  Multi-purpose Smart Robot"
+---
+
+# Project Name
+
+  Multi-purpose Smart Robot
+
+## Summary
+
+  I am going to build a mini robot which can solve questions, make decisions, manage digital activities and serve as a virtual assistant.
+  
+  It is very useful since it can be used to solve my studies related problems and it can create a customized planner for me.
+  
+  I am very excited to build this project.
+
+  It can detect objects, search images, manage app time and their activities for other devices, serve as AI Assistant ad can tell climatic conditions.
```

Comment:
could you please elaborate on a few of these features you'll be implementing and how you'll be going about it? please include that as part of your plan :)) thank you sm!

Reply:
I will be using it as a voice assistant. It could be used for object detection in order to classify objects.
It will manage my screen time by Pomodoro technique and It could search image with image search engine.
It will also have sensors which could detect the climatic condition.
I am also planning to use it for Nature or Wildlife Camera as it has the Tensorflow API which can detect the species of  creatures very easily.